Data Streams: Add ecs@mappings disclaimer for index templates cloned on ES < 8.13 (#1174)
elastic/ingest-docs: docs/en/ingest-management/data-streams.asciidoc

**Goal:** Customize the {ilm-init} policy for the `system.network` data stream in the `production` namespace.
Specifically, apply the built-in `90-days-default` {ilm-init} policy so that data is deleted after 90 days.

> NOTE: If you cloned an index template to customize the data retention policy on an {es} version prior to 8.13, you must update the index template in the clone to use the `ecs@mappings` component template on {es} version 8.13 or later. See "Update index template cloned before {es} 8.13" section for the step-by-step instructions.

Reviewer: ```suggestion
NOTE: If you cloned an index template to customize the data retention policy on an {es} version prior to 8.13, you must update the index template in the clone to use the `ecs@mappings` component template on {es} version 8.13 or later. See <<data-streams-pipeline-update-cloned-template-before-8.13,Update index template cloned before {es} 8.13>> for the step-by-step instructions.
```
This will add a link to the new section.

POST /metrics-system.network-production/_rollover/
----

[discrete]
[[data-streams-pipeline-update-cloned-template-before-8.13]]
== Update index template cloned before {es} 8.13

> If you cloned an index template to customize the data retention policy on an {es} version prior to 8.13, you must update the index cloned index template adding the `ecs@mappings` component template on {es} version 8.13 or later.

Reviewer: ```suggestion
If you cloned an index template to customize the data retention policy on an {es} version prior to 8.13, you must update the index cloned index template to add the `ecs@mappings` component template on {es} version 8.13 or later.
```

> Here how you can update the cloned index template:

Reviewer: ```suggestion
To update the cloned index template:
```
